### ISSUE TITLE
Conflicting information about the presence of an official CLI client in the documentation

### ISSUE THREAD

**Issue by phao5814** [dokku/dokku#4235]
In the [documentation](http://dokku.viewdocs.io/dokku/community/clients/), it says the following:

> **While Dokku does not yet have an official client**, there are a multitude of ways in which you can interact with your Dokku installation. The easiest is to use the bash client, though you may wish to use another.

However, just below that paragraph, there is the following:

> OFFICIAL CLIENT
> See the remote commands documentation for more information on how to install and **use the official client.**

Furthermore, there is additional [documentation](http://dokku.viewdocs.io/dokku/deployment/remote-commands/#official-client) for "the official client".

Not sure if this inconsistency is intentional but it is confusing me a bit... is there an official Dokku client?

**Comment by josegonzalez** [comment]
Hmm I guess there is an "official" client.

Might be worth spitting it out into it's own "package" for various OS systems. Thoughts?

**Comment by phao5814** [comment]
Thanks for confirming that there is an official client :) Given this is the case, it would probably be best to amend the documentation so that it doesn't say that "Dokku does not yet have an official client".

With respect to how this client is distributed, splitting it out into its own package for different OS systems seems fairly reasonable. In my case, since I used MacOS, I just ran the brew command `brew install dokku/repo/dokku` outlined in the [documentation](http://dokku.viewdocs.io/dokku/deployment/remote-commands/#installation-via-homebrew).
